desktop/core/src/desktop/js/reactComponents/MetricsComponent/MetricsComponent.test.tsx

```diff
@@ -96,4 +96,15 @@ describe('MetricsComponent', () => {
       });
     }
   });
+
+  // Test for ensuring metrics starting with 'python.gc', 'multiprocessing', and 'auth' are not displayed
```

Comment:
Nit: Comment is superfluous

Reply:
@JohanAhlen 
I just added it to maintain consistency because the other 2 tests in the same file have comment telling what the test is about.
Should I keep it or still remove it?